release/models/keychain/openconfig-keychain.yang

```diff
@@ -194,13 +200,19 @@ module openconfig-keychain {
 
     leaf key-id {
       type union {
-        type oc-yang:hex-string {
+        type oc-yang:hex-string-prefixed {
+          length "3..66";
+        }
+       type oc-yang:hex-string {
```

Comment:
Suggest to remove this, since otherwise, we'll fall through and "123" will still be parsed as a `hex-string`.